### PR TITLE
Translate CVE-2019-16201 post (tr)

### PR DESCRIPTION
**Pull request by ismailarilik**

### CODE REVIEW

File: tr/news/_posts/2019-10-01-webrick-regexp-digestauth-dos-cve-2019-16201.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: "CVE-2019-16201: WEBrick'in Özet erişim kimlik doğrulamasında Düzenli İfade Hizmet Reddi güvenlik açığı"
+author: "mame"
+translator: "İsmail Arılık"
+date: 2019-10-01 11:00:00 +0000
+tags: security
+lang: tr
+---
+
+WEBrick'in Özet kimlik doğrulama modülünde düzenli ifade hizmet reddi güvenlik açığı bulundu.
+Bir saldırfgan bu açığı, bir WEBrick servisine karşı etkili bir hizmet reddi yapmak için kullanabilir.
```
[the remark this answers is not in the document]
Comment (ismailarilik) [reply tr/news/_posts/2019-10-01-webrick-regexp-digestauth-dos-cve-2019-16201.md]: e6ee2a2

**Comment by stomar** [comment]
Please use shorter commit messages to avoid truncation; date and/or CVE number would be sufficient, e.g. „Translate CVE-2019-16201 post (tr)“.

**Comment by ismailarilik** [comment]
@enderahmetyurt

**Comment by stomar** [comment]
Also, commit messages of squashed typo fixes etc. can be omitted in the final commit, it's more confusing than helpful. E.g.

```
Translate foo post (tr)

* Translate foo post (tr)
* Fix typo
```

doesn't make sense in the commit history. Simply "Translate foo post (tr)" would be clearer.

**Pull request merged**